### PR TITLE
Add settings option to toggle Minecraft news display.

### PR DESCRIPTION
**Pull request by TOLoneWolf**

### CODE REVIEW

File: src/app/desktop/components/News.js [hunk 1]
```diff
@@ -200,38 +201,44 @@ function useInterval(callback, delay) {
...
+          height="158"
+          css={`
+            border-radius: 2px;
+          `}
+        />
+      </StyledContentLoader>
+    );
+    // eslint-disable-next-line no-else-return
+  } else {
+    return '';
```
Comment (blarfoon) [inline src/app/desktop/components/News.js]: Don't return `''`

File: src/app/desktop/components/News.js [hunk 2]
```diff
@@ -200,38 +201,44 @@ function useInterval(callback, delay) {
...
+          ry="0"
+          width="830"
+          height="158"
+          css={`
+            border-radius: 2px;
+          `}
+        />
+      </StyledContentLoader>
+    );
+    // eslint-disable-next-line no-else-return
```
Comment (blarfoon) [inline src/app/desktop/components/News.js]: There's no need for the `else`

File: src/app/desktop/components/News.js [hunk 3]
```diff
@@ -200,38 +201,44 @@ function useInterval(callback, delay) {
...
+          rx="0"
+          ry="0"
+          width="830"
+          height="158"
+          css={`
+            border-radius: 2px;
+          `}
+        />
+      </StyledContentLoader>
+    );
```
Comment (blarfoon) [inline src/app/desktop/components/News.js]: What I would do, is add a `if(!showNews) return null` before the return of the news and don't change anything else

**Review by blarfoon — CHANGES_REQUESTED** [review]

File: src/app/desktop/components/News.js (hunk 1 above)
Comment (TOLoneWolf) [reply src/app/desktop/components/News.js]: ah okay.
Was thinking it was similar to state and needed a return.

**Review by blarfoon — APPROVED** [review]

**Pull request merged**